### PR TITLE
Feat - Allow admin to add user experience on event talks

### PR DESCRIPTION
**Pull request by JonasTeixeira42**
This PR solves the issue #411 

- It creates the new talking experience register page for active admin
image

### CODE REVIEW

File: app/models/talking.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Talking < ApplicationRecord
```
Comment (adbatista) [inline app/models/talking.rb:3]: Try to add the model name of a resource instead an action; actions should be the name of functions or commands.

A proper name here would be `Talk` or Event

File: app/models/talking.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  validates :event_name, :talk_title, :date, presence: true
+  validate :future_date?
+
+  private
+
+  def future_date?
+    return if date.blank?
+
+    errors.add(:date, "must be before the today's date") if date.future?
```
Comment (adbatista) [inline app/models/talking.rb:14]: Add the translation in the i18n here; active record already provides an approach for this

File: spec/admin/talking_spec.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Talking do
```
Comment (adbatista) [inline spec/admin/talking_spec.rb:5]: nice

File: app/models/talking.rb (hunk 1 above)
Comment (JonasTeixeira42) [reply app/models/talking.rb:3]: on it

File: app/models/talk.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  validates :event_name, :talk_title, :date, presence: true
+  validate :future_date?
+
+  private
+
+  def future_date?
+    return if date.blank?
+
+    errors.add(:date, I18n.t('activerecord.errors.models.talk.attributes.date.future_date')) if date.future?
```
Comment (adbatista) [inline app/models/talk.rb:14]: use a type instead adding the call to i18n here, Active record already do this https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add

Comment (JonasTeixeira42) [reply app/models/talk.rb:14]: on it

**Review by adbatista — APPROVED** [review]

**Pull request merged**